pkg/cmdlets/errors/errors.go

```diff
@@ -117,6 +108,37 @@ func Unwrap(err error) (int, error) {
 		}
 	}
 
+	if hasMarshaller {
+		return code, err
+	}
+
+	return code, &OutputError{err}
+}
+
+func ReportError(err error, flags []*captain.Flag, an analytics.Dispatcher) {
+	var ee errs.Errorable
+	stack := "not provided"
+	isErrs := errors.As(err, &ee)
+	if isErrs {
+		stack = ee.Stack().String()
+	}
+
+	_, hasMarshaller := err.(output.Marshaller)
+
+	// Log error if this isn't a user input error
+	if !locale.IsInputError(err) {
+		multilog.Critical("Returning error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
+	} else {
+		var flagNames []string
+		for _, flag := range flags {
+			flagNames = append(flagNames, fmt.Sprintf("--%s", flag.Name))
+		}
+		logging.Debug("Returning input error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
+		an.Event(anaConst.CatDebug, anaConst.ActInputError, &dimensions.Values{
+			Trigger: p.StrP(strings.Join(flagNames, ", ")),
```

Comment:
This should contain the command as well, not just the flags.

eg. "state --version", not just "--version".